### PR TITLE
Network examples

### PR DESCRIPTION
**Pull request by gijsio**
Made changes to network examples
added antenna overview

### CODE REVIEW

File: content/tutorials/networkprotocols/mdns.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "MDNS"
+---
+On this page, we will explore the Multicast DNS feature. MDNS is very useful for connecting to a device of which you do not know the IP address, and works similar to the Domain Name System, on a local network without the need for a DNS server. You can use the `hostname.local` url to connect. To be able to use MDNS on your computer, you might need some additional software.
+* Windows:
+* MacOS: built in!
+* Linux:
```
Comment (peter-pycom) [inline content/tutorials/networkprotocols/mdns.md:7]: e.g., mdns-scan, avahi-resolve-host-name

File: content/tutorials/networks/lora/_index.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ aliases:
 The following tutorials demonstrate the use of the LoRa functionality on the LoPy. LoRa can work in 2 different modes; **LoRa-MAC** (which we also call Raw-LoRa) and **LoRaWAN** mode.
 > Note: LoRa will not work with Pygate firmware loaded on your device. It will return a `/event_groups.c:498 (xEventGroupClearBits)- assert failed!` error.
 
+When using Lora, **Always** connect the appropriate LoRa antenna to your device. See the figures below for the correct antenna placement
+
+| Lopy / Fipy | Lopy4 |
+|:---|:---|
+| ![](/gitbook/assets/lora_pigtail_lopy.png) | ![](/gitbook/assets/lora_sigfox_pigtail_lopy4.png) |
+
```
Comment (peter-pycom) [inline content/tutorials/networks/lora/_index.md:14]: thanks :)

File: content/tutorials/networks/wlan.md [hunk 3]
```diff
@@ -6,13 +6,19 @@ aliases:
     - chapter/tutorials/networks/wlan
 ---
 
-The WLAN (WiFi) is a system feature of all Pycom devices, therefore it is enabled by default.
+The WLAN (WiFi) is a system feature of all Pycom devices, therefore it is enabled by default. The development boards include an on-board antenna by default, so no external antenna is needed to get started. When deploying your solution, you might want to concider using the external antenna to increase the wireless range.
```
Comment (peter-pycom) [inline content/tutorials/networks/wlan.md:10]: concider -> consider

File: content/firmwareapi/pycom/network/wlan.md [hunk 4]
```diff
@@ -218,24 +218,11 @@ Each element of the returned list is a tuple, containing the MAC address and IP
...
-* [68, 73]: level3
-* [60, 67]: level4
-* [52, 59]: level5
-* [44, 51]: level5 - 2dBm
-* [34, 43]: level5 - 4.5dBm
-* [28, 33]: level5 - 6dBm
-* [20, 27]: level5 - 8dBm
-* [8, 19]: level5 - 11dBm
-* [-128, 7]: level5 - 14dBm
+Values passed in power are mapped to transmit power levels as power = [8, 78] = [2, 20] dBm, in 0.25dBm increments.
```
Comment (peter-pycom) [inline content/firmwareapi/pycom/network/wlan.md]: I agree it was confusing, but I still find it confusing. maybe just use more words

the possible values are from 8 to 78. this maps to dBm value in the range bla, so max_tx_pwr=8 means 2 dBm, max_tx_pwr = 68 means blub, yadayada 0.25 dBm increments

**Review by peter-pycom — CHANGES_REQUESTED** [review]

File: content/firmwareapi/pycom/network/wlan.md (hunk 4 above)
Comment (gijsio) [reply content/firmwareapi/pycom/network/wlan.md]: I improved the situation

**Review by peter-pycom — APPROVED** [review]

**Pull request merged**